### DISCUSSION TITLE
Installing Argocd on GKE 1.28

### DISCUSSION THREAD

**Discussion by vini-intenseye** [argoproj/argo-cd#17402] [Q&A]
i have installed argocd version `v2.10.2+fcf5d8c`  on GKE version `v1.27.8-gke.1067004` . I m trying to do the same install on GKE version `v1.28.5-gke.1217000` and i get the following error

```
Failed to load live state: failed to get cluster info for "https://kubernetes.default.svc": error synchronizing cache state : error getting openapi resources: the server is currently unable to handle the request

Failed to load target state: failed to get cluster version for cluster "https://kubernetes.default.svc": failed to get cluster info for "https://kubernetes.default.svc": error synchronizing cache state : error getting openapi resources: the server is currently unable to handle the request

error synchronizing cache state : error getting openapi resources: the server is currently unable to handle the request
```

I have no changes on the install files. Has anyone successfully deployed argocd on GKE 1.28? Is there any configuration required for it to work? Thanks in advance

**Comment by yuehaii** [comment]
Not sure whether it is related https://cloud.google.com/kubernetes-engine/docs/release-notes#February_06_2024.  
You can try GKE 1.28.7-gke.1026000 to see whether it helps.
